browse/templates/base.html

```diff
@@ -38,10 +38,17 @@
   <script language="javascript" src="//code.jquery.com/jquery-latest.min.js"></script>
   <script language="javascript" src="//static.arxiv.org/js/user_banner.js"></script>
   {% endif -%}
+  <style type="text/css">
+    @media screen and (max-width: 1023px) {
```

Comment:
hold for update to the width on this number, I have to reconcile this with other pages (it forces columns on footer)

Reply:
yep, this rule that I added should now be removed. It's superseded in the arxiv-ARXIVNG-1918.css stylesheet.